apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -0,0 +1,365 @@
+      sorting: false,
+      removable: false,
+      field: 'rowActionButtons',
+    },
+    {
+      title: 'Proposal ID',
+      field: 'proposalId',
+    },
+    {
+      title: 'Title',
+      field: 'title',
+      ...{ width: 'auto' },
+    },
+    {
+      title: 'Principal Investigator',
+      field: 'principalInvestigator',
+      render: (proposalView: ProposalViewData) => {
+        if (proposalView.principalInvestigator?.lastname) {
+          if (proposalView.principalInvestigator?.preferredname) {
+            return `${proposalView.principalInvestigator.lastname}, ${proposalView.principalInvestigator.preferredname}`;
+          } else if (proposalView.principalInvestigator?.firstname) {
+            return `${proposalView.principalInvestigator.lastname}, ${proposalView.principalInvestigator.firstname}`;
+          }
+        }
+
+        return '';
+      },
+      customFilterAndSearch: () => true,
+    },
+    {
+      title: 'PI Email',
+      field: 'principalInvestigator.email',
+      sorting: false,
+      emptyValue: '-',
+    },
+    {
+      title: 'Status',
+      field: 'statusName',
+    },
+    {
+      title: 'Date submitted',
+      field: 'submittedDate',
+      ...{ width: 'auto' },
+    },
+  ];
+
+  const instrumentManagementColumns = (
+    t: TFunction<'translation', undefined>
+  ) => [
+    {
+      title: t('instrument'),
+      field: 'instruments.name',
+      render: (rowData: ProposalViewData) =>
+        fromArrayToCommaSeparated(
+          rowData.instruments?.map((instrument) => instrument.name)
+        ),
+      customFilterAndSearch: () => true,
+    },
+  ];
+
+  const techniquesColumns = () => [
+    {
+      title: 'Technique',
+      field: 'technique.name',
+      render: (rowData: ProposalViewData) =>
+        fromArrayToCommaSeparated(
+          rowData.techniques?.map((technique) => technique.name)
+        ),
+      customFilterAndSearch: () => true,
+    },
+  ];
+
+  addColumns(columns, instrumentManagementColumns(t));
+  addColumns(columns, techniquesColumns());
+
+  columns = setSortDirectionOnSortField(columns, sortField, sortDirection);
+
+  const { loading, proposalsData, totalCount } = useProposalsCoreData(
+    {
+      proposalStatusId: proposalFilter.proposalStatusId,
+      techniqueFilter: proposalFilter.techniqueFilter,
+      instrumentFilter: proposalFilter.instrumentFilter,
+      callId: proposalFilter.callId,
+      referenceNumbers: proposalFilter.referenceNumbers,
+      dateFilter: proposalFilter.dateFilter,
+      text: queryParameters.searchText,
+      excludeProposalStatusIds: proposalFilter.excludeProposalStatusIds,
+    },
+    queryParameters.query
+  );
+
+  const [tableData, setTableData] = useState<ProposalViewData[]>([]);
+  const [preselectedProposalsData, setPreselectedProposalsData] = useState<
+    ProposalViewData[]
+  >([]);
+
+  const { instruments, loadingInstruments } = useXpressInstrumentsData(
+    proposalsData,
+    techniques
+  );
```

Comment:
Is this query also necessary because if we already have techniques and calls with instruments we can still get instruments related to the technique and call or filter them with calls on proposals.